javascript/react_js/advanced_concepts.md

```diff
@@ -26,7 +26,7 @@ By the end of this lesson, you should be able to do the following:
 
 #### <span id="prop-types">1. PropTypes</span>
 
-One common discussion about JavaScript is whether it would be better if you could declare types for variables and properties. Many programmers agree that the pattern of declaring types, which you have in many other programming languages is preferable because it allows you to catch errors, such as passing a string to a variable that should be a number. Therefore, React provides the possibility to declare types using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html). And if that's not enough for you, you could also use [TypeScript](https://www.typescriptlang.org/) with React.
+One common discussion about JavaScript is whether it would be better if you could declare types for variables and properties. Many programmers agree that the pattern of declaring types, which you have in many other programming languages is preferable because it allows you to catch errors, such as passing a string to a variable that should be a number. Therefore, React provides the possibility to declare types using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html). But it is not used in modern react anymore, you should use [TypeScript](https://www.typescriptlang.org/) with React for static type checking.
```

Comment:
So checking the type checking page. It recommends using this instead of typescript https://www.npmjs.com/package/prop-types. In this case I think it would be best if we simply replaced the prototypes link with the reccomended package instead of saying to use TypeScript with React as the go to. specifically change "React provides the possibility to declare types using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html)" to  "React recommends using the [prop-types](https://www.npmjs.com/package/prop-types) for runtime type checking for React props and similar objects."

Then keep the last sentence as "if that's not enough for you, you could also use [TypeScript](https://www.typescriptlang.org/) with React."

Reply:
Hello @linkonsat I have done the desired changes in my latest commits. Please review it and let me know if I need to do further changes.